### PR TITLE
Fixes #29590 - Katello report template for listing host packages

### PR DESCRIPTION
**Pull request by ianballou**
Adds a new Katello report template for listing a host's installed packages. Includes a search filter for limiting the targeted hosts.

### CODE REVIEW

File: app/views/unattended/report_templates/host_-_installed_packages [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<%#
+name: Host - All Installed Packages
+snippet: false
+template_inputs:
+- name: Hosts filter
+  required: false
+  input_type: user
+  description: Limit the report only to hosts found by this search query. Keep empty
```
Comment (chris1984) [inline app/views/unattended/report_templates/host_-_installed_packages]: Looks good, one small change:

`Limit the report to only hosts found by this search query`

**Review by chris1984 — DISMISSED** [review]
Ack, one small comment

**Review by chris1984 — APPROVED** [review]
LGTM

**Comment by chris1984** [comment]
@jeremylenz can this get a merge?

**Review by jeremylenz — APPROVED** [review]

**Pull request merged**